LICENSE

```diff
@@ -19,3 +43,58 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+-------------------------------------------------------------------------------
+
+PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
+
+1. This LICENSE AGREEMENT is between the Python Software Foundation
```

Comment:
Why not just add the link to the licence file just like Apache license?

https://opensource.org/licenses/Python-2.0

Reply:
We had the choice to do either. The link to Apache license is because Apache 2.0 license is rather long. In this case, since both MIT and PSF are short, its easier on the reader to reproduce the entire license here so as to do one less look-up.